getting_started/first_3d_game/09.adding_animations.rst

```diff
@@ -82,6 +82,11 @@ the timeline.
 Let's insert our first keys. Here, we will animate both the position and the
 rotation of the ``Character`` node.
 
+.. warning::
+    It is important to create keyframes from the ``Character`` node and not the root
+    ``Player`` node. Using the root of the player scene causes animations to override
+    user input and will prevent the player from being able to move.
```

Comment:
I don't think this warning should be added. The sentence below this explicitly says to select "Character" and how to create the keyframes. I understand how the screenshots caused confusion. But with those fixed now in this PR this warning isn't necessary.

Reply:
I agree that with the screenshots updates, this is a slightly less relevant comment, however, it also highlights an important concept about *why* it's important to build keyframes from that node as opposed to the root of the scene. Perhaps instead of a warning, this could be reframed and placed inline with the introduction?

Reply:
I'm removing the warning for now, but I do think that understanding the reason for _why_ choosing the character node is important could be valuable as part of the animation section.